support/remote_health_monitoring/enabling-remote-health-reporting.adoc

```diff
@@ -0,0 +1,19 @@
+[id="enabling-remote-health-reporting"]
+= Enabling remote health reporting
+include::modules/common-attributes.adoc[]
+
+:context: enabling-remote-health-reporting
+
+toc::[]
+
+
+If you or your organization has disabled remote health reporting, it is possible to enable this feature again. You can see that remote health reporting is disabled from the message "Insights not available" in the *Status* tile on the {product-title} Web Console Overview page. 
```

Comment:
"have" instead of "has".
"you can enable this feature again" instead of "it is possible..."